lagunita/lms/static/css/vendor/su-identity.css

```diff
@@ -1,291 +1,231 @@
+/*
+========================================================
+Custom base styles
+*/
+
 #su-header .su-container,
 #su-footer .su-container {
   margin: 0 auto;
 }
-
```

Comment:
Generally speaking, please split out whitespace changes into a separate PR, or at the very least, a separate commit. It makes this code significantly more difficult to review, as close to 50% of the changes are cosmetic.